### PR TITLE
[NTOS:CC][NTOS:MM] 2 functions: Fix Iosb annotation

### PR DESCRIPTION
**Pull request by SergeGautherie**
Noticed while checking #3679.

### CODE REVIEW

File: ntoskrnl/cc/view.c [hunk 1]
```diff
@@ -158,7 +158,7 @@ CcRosTraceCacheMap (
 NTSTATUS
 CcRosFlushVacb (
     _In_ PROS_VACB Vacb,
-    _In_ PIO_STATUS_BLOCK Iosb)
+    _Out_opt_ PIO_STATUS_BLOCK Iosb)
```
Comment (binarymaster) [inline ntoskrnl/cc/view.c:161]: Show me where in this function `Iosb` parameter gets written into.

**Review by binarymaster — DISMISSED** [review]

File: ntoskrnl/cc/view.c (hunk 1 above)
Comment (SergeGautherie) [reply ntoskrnl/cc/view.c:161]: Really?
https://github.com/reactos/reactos/blob/c3dbcde1e45ab4c5c2d977a92fda1160bb7571e6/ntoskrnl/cc/view.c#L178-L181

Comment (binarymaster) [reply ntoskrnl/cc/view.c:161]: For some reason I don't think it's a write operation.

Comment (SergeGautherie) [reply ntoskrnl/cc/view.c:161]: What do you think it is, then?

Comment (GeoB99) [reply ntoskrnl/cc/view.c:161]: `MmFlushSegment` writes data into the I/O status block the information field and status code respectively, only if such block is actually filled to the call. 
https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/mm/section.c;hb=d07203ecb700a11c21608448d2326ea6a6b3f368#l4662

In the case of `CcRosFlushVacb` we write back to the I/O status block whatever stuff and the caller can do whatever it wants with this variable after it's been written to it. With that said it'd certainly make sense to treat this argument as `_Out_`.

Comment (binarymaster) [reply ntoskrnl/cc/view.c:161]: That was not clear for me, thanks for the explanation. :+1:

**Review by GeoB99 — APPROVED** [review]
LGTM but take my review with a slight grain of salt regarding whether a I/O status block is mandatory on return when flushing the VACB structure or not. Maybe @zefklop can help us deciding that.

**Review by tkreuzer — APPROVED** [review]

**Pull request merged**